src/MetaMorpho.sol

```diff
@@ -772,7 +773,13 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             if (supplyCap == 0) continue;
 
             MarketParams memory marketParams = _marketParams(id);
-            (uint256 supplyAssets,,) = _accruedSupplyBalance(marketParams, id);
```

Comment:
should this function be renamed?

Reply:
why here we don't use the morpho balance lib?

Reply:
Which one ?

Reply:
because we round up in this contract while morpho balance lib rounds down

Reply:
I meant why we were not using morpho lib